docs/analytics-ga4/ga4-focus-loss-tracker.md

```diff
@@ -0,0 +1,16 @@
+# Google Analytics 4 focus loss tracker
```

Comment:
Good documentation. Can you include a link to it in `ga4-all-trackers.md`?

Reply:
Thanks, done 👍